Casks/5/5kplayer.rb

```diff
@@ -4,14 +4,21 @@
 
   url "https://www.5kplayer.com/download/5kplayer.dmg"
   name "5KPlayer"
-  desc "Play 4K/1080p/360 degree video, MP3 AAC APE FLAC music without quality loss"
+  desc "Play 4K/1080p/360p degree video, MP3/AAC/APE/FLAC music without quality loss"
```

Comment:
```suggestion
  desc "Play 4K/1080p/360 degree video, MP3/AAC/APE/FLAC music without quality loss"
```

The 360 is in relation to 360° video, not 360p resolution.

Reply:
Shouldn't it rather be hyphenated?

```suggestion
  desc "Play 4K/1080p/360-degree video, MP3/AAC/APE/FLAC music without quality loss"
```